themes/classic/templates/checkout/_partials/cart-voucher.tpl

```diff
@@ -53,19 +53,18 @@
             <form action="{$urls.pages.cart}" data-link-action="add-voucher" method="post">
               <input type="hidden" name="token" value="{$static_token}">
               <input type="hidden" name="addDiscount" value="1">
-              <input class="promo-input" type="text" name="discount_name" placeholder="{l s='Promo code' d='Shop.Theme.Checkout'}">
-              <button type="submit" class="btn btn-primary"><span>{l s='Add' d='Shop.Theme.Actions'}</span></button>
+              <input class="promo-input" type="text" name="discount_name" placeholder="{l s='Promo code' d='Shop.Theme.Checkout'}"><!--
```

Comment:
useless comment? :thinking:

Reply:
Nope. it's use to make theses two element closer with no space between. No need css to do that

Reply:
Add a comment to explain or add both elements in one line.

Reply:
I commented it